src/hoki.js

```diff
@@ -68,33 +50,24 @@ function observer(event, callback) {
         throw new TypeError('Callback must be a function');
     }
 
-    if (!events[event]) {
-        return;
-    }
-
-    events[event].push(callback);
+    if (events[event]) events[event].push(callback);
 }
 
-// Dispatch events with or without data
+// Dispatch events
 function dispatcher(event, data) {
-    if (!isString(event)) {
-        throw new TypeError('Event must be a string');
-    }
-
-    if (!events[event]) {
-        return;
-    }
-
-    events[event].forEach(callback => (data ? callback(data) : callback()));
+    if (!isString(event)) throw new TypeError('Event must be a string');
+    
+    // call every callback in event list
+    if (events[event]) events[event].forEach(cb => cb(data));
 }
 
 // Return events available
 const list = () => Object.keys(events);
 
 module.exports = {
-    register: register,
-    unregister: unregister,
-    dispatcher: dispatcher,
-    observer: observer,
-    events: list,
-};
+    register,
+    unregister,
+    dispatcher,
+    observer,
+    events: list
```

Comment:
What I ment was renaming const events = {} to something else, then we could keep the export clean :)

Reply:
ah ok sure...